testing/main_test.go

```diff
@@ -193,11 +193,6 @@ var minRequiredVersionsByORMName = map[string]struct {
 	},
 }
 
-// minTenantVersion is the minimum version that supports creating SQL tenants
-// (i.e. the `cockroach mt start-sql command). Earlier versions cannot create
-// tenants.
-var minTenantVersion = version.MustParse("v20.2.0-alpha")
```

Comment:
is there any chance we could just change this to `v20.2.0-alpha.3` and call it a day? The existing comparison logic in `version.AtLeast` would handle that correctly.

Failing that, I think it would still be preferable to take the version returned from `SHOW CLUSTER SETTING version` and put it into a `version.Version` struct, so that the version comparison logic is just in one place.

Reply:
No, `20.2.0-alpha.3` corresponds to `20.1-16`, we need at least `-18` (though if we discuss this for much longer, I might be able to put the 20.2 RCs in here... :-) ). I don't think your other suggestion is as clean-cut as you imagine, or I'm missing something. What do I put in that `version.Version`? I can put `20.1` and where do I put the `-17`? We will be comparing apples and oranges and I'm not quite sure why; my desire to find out in which order the `-PRERELEASE+METADATA` stuff compares is limited and I can't imagine it'll be clearer to those who come after me. Or are you suggesting that I refactor everything to source the versions from the cluster version? That could be done - my motivation is limited - but it could be done.

Reply:
understood, then i won't hold this up any longer :)

no i wasn't expecting a full refactor for everything to use the cluster version. i thought you could have something like

```
minClusterVersion = version.MustParse("v20.1-18")

// later
db.QueryRow(`SHOW CLUSTER SETTING version`).Scan(&clusterVersion)
crdbClusterVerion = version.MustParse(clusterVersion)

if crdbClusterVersion.AtLeast(minClusterVersion) {
  // ...
}
```

this would involve changing the regexp for a valid version so that the `patch` number is optional.

anyway, i acknowledge now that this approach ultimately could be more confusing, so lgtm as is. thanks again for the fix.